Dockerfile.conda

```diff
@@ -0,0 +1,76 @@
+# Set the base image
+FROM continuumio/miniconda3:latest AS builder
+
+# Install system dependencies
+RUN apt-get update && \
+    apt-get install -y sudo libusb-1.0 gcc g++ python3-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /home/hummingbot
+
+# Create conda environment
+COPY setup/environment-simplified.yml /tmp/environment.yml
+RUN conda env create -f /tmp/environment.yml && \
+    conda clean -afy && \
+    rm /tmp/environment.yml
+
+# Copy remaining files
+COPY bin/ bin/
+COPY hummingbot/ hummingbot/
+COPY scripts/ scripts/
+COPY setup.py .
+COPY LICENSE .
+COPY README.md .
+COPY DATA_COLLECTION.md .
+
+# activate hummingbot env when entering the CT
+SHELL [ "/bin/bash", "-lc" ]
+RUN echo "conda activate hummingbot" >> ~/.bashrc
+
+RUN python3 setup.py build_ext --inplace -j 8 && \
+    rm -rf build/ && \
+    find . -type f -name "*.cpp" -delete
+
+
+# Build final image using artifacts from builder
+FROM continuumio/miniconda3:latest AS release
+
+# Dockerfile author / maintainer
+LABEL maintainer="Fede Cardoso @dardonacci <federico@hummingbot.org>"
+
+# Build arguments
+ARG BRANCH=""
+ARG COMMIT=""
+ARG BUILD_DATE=""
+LABEL branch=${BRANCH}
+LABEL commit=${COMMIT}
+LABEL date=${BUILD_DATE}
+
+# Set ENV variables
+ENV COMMIT_SHA=${COMMIT}
+ENV COMMIT_BRANCH=${BRANCH}
+ENV BUILD_DATE=${DATE}
+
+ENV INSTALLATION_TYPE=docker
+
+# Install system dependencies
+RUN apt-get update && \
+    apt-get install -y sudo libusb-1.0 && \
+    rm -rf /var/lib/apt/lists/*
+
+# Create mount points
+RUN mkdir -p /home/hummingbot/conf /home/hummingbot/conf/connectors /home/hummingbot/logs /home/hummingbot/data /home/hummingbot/certs /home/hummingbot/scripts
```

Comment:
Is a mount point needed for `/home/hummingbot/conf/strategies`?

Reply:
I will need to do some tests but probably can be there. I think that probably is not necessary since you don't have it at the start but I can check